src/Acuminator/Acuminator.Analyzers/StaticAnalysis/Descriptors.cs

```diff
@@ -406,6 +406,12 @@ private static DiagnosticDescriptor Rule(string id, LocalizableString title, Cat
 		public static DiagnosticDescriptor PX1076_CallToPXInternalUseOnlyAPI_OnlyISV { get; } =
 			Rule("PX1076", nameof(Resources.PX1076Title).GetLocalized(), Category.Acuminator, DiagnosticSeverity.Warning, DiagnosticsShortName.PX1076);
 
+		public static DiagnosticDescriptor PX1077_EventHandlersShouldBeProtectedVirtual { get; } =
```

Comment:
@taskinkemal  private events should be still reported as Errors, so a separate descriptor is required for this case.

Reply:
Done.